components/molecules/dropdown/components/menu.js

```diff
@@ -4,27 +4,30 @@
 
 // React
 import React from 'react'
-import { array, func, string } from 'prop-types'
+import { array, func, oneOf } from 'prop-types'
 
 // Style
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 
 // UI
+import { elementTypes } from 'index'
 import { DropdownItem } from '../../'
+import { Position } from '../../../'
```

Comment:
@mhsnprvr2  Same issues as other comments - focus on the details

import { elementTypes } from 'index'
import { DropdownItem } from '../../'	import { DropdownItem } from '../../'
import { Position } from '../../../'

Could be moved into one import string I am sure